### ISSUE TITLE
Stand-alone App (Windows) doesn't work

### ISSUE THREAD

**Issue by fireattack** [apprenticeharper/DeDRM_tools#512]
It says ""ImportError: No module named calibre_plugins.dedrm".

```
F:\DeDRM_App\DeDRM_lib>py -2 DeDRM_App.pyw B076HLXHM7_EBOK.azw
Using Library AlfCrypto DLL/DYLIB/SO
Traceback (most recent call last):
  File "DeDRM_App.pyw", line 66, in <module>
    from scriptinterface import decryptepub, decryptpdb, decryptpdf, decryptk4mo
bi
  File "F:\DeDRM_App\DeDRM_lib\lib\scriptinterface.py", line 14, in <module>
    import k4mobidedrm
  File "F:\DeDRM_App\DeDRM_lib\lib\k4mobidedrm.py", line 93, in <module>
    import kfxdedrm
  File "F:\DeDRM_App\DeDRM_lib\lib\kfxdedrm.py", line 20, in <module>
    from calibre_plugins.dedrm import ion
ImportError: No module named calibre_plugins.dedrm

F:\DeDRM_App\DeDRM_lib>
```

**Comment by fireattack** [comment]
After some debugging I found that the problem is caused by the lack of `backports.lzma` that is imported in `ion.py`. 

The problem is gone after I installed it (I did have some trouble to install that package on Windows due to https://github.com/peterjc/backports.lzma/issues/22, but that's another story). 

Thanks!

**Comment by ElleKayEm** [comment]
Looks like the stand-alone Mac app is having a similar issue #505.

**Comment by fireattack** [comment]
@ElleKayEm Yeah. Also #482

**Comment by frioux** [comment]
I don't think this should be closed.  If the tools depend on ion.py, the docs don't clearly say so or how to install it.

**Comment by j-howell** [comment]
The ion.py module is included in the tools in order to support Amazon Kin dle KFX format. The problem is that ion.py imports lzma and and if that is not present the fallback handling produces a misleading error message.

It appears that a fix has already been made (#489) and should be in the next release of the tools.

**Comment by frioux** [comment]
Got it. Thanks for the clarification

--
